sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -271,6 +272,28 @@ export class AssessmentController {
           );
           return job.complete();
         }
+        const application = assessment.application;
+        if (
+          application.applicationStatus === ApplicationStatus.Completed &&
+          !assessment.previousDateChangedReportedAssessment
+        ) {
```

Comment:
Please add a comment explaining why `!assessment.previousDateChangedReportedAssessment` is added?